File: app/src/main/java/igrek/forceawaken/service/time/AlarmTimeService.java
package igrek.forceawaken.service.time;

import org.joda.time.DateTime;

import java.util.Random;

public class AlarmTimeService {
	
	private Random random = new Random();
	
	public AlarmTimeService() {
	}
	
	public DateTime getFakeCurrentTime() {
		// 2 hours forward
		return DateTime.now().plusMinutes(random.nextInt(2 * 60));
	}
}

File: app/src/main/java/igrek/forceawaken/time/AlarmTimeService.kt
package igrek.forceawaken.time

import org.joda.time.DateTime
import java.util.*

class AlarmTimeService {
    private val random = Random()

    // 2 hours forward
    val fakeCurrentTime: DateTime
        get() =// 2 hours forward
            DateTime.now().plusMinutes(random.nextInt(2 * 60))
}